source/default/_patterns/02-molecules/tabs/index.js

```diff
@@ -0,0 +1,125 @@
+
+  const deactivatePanels = ($panels) => {
+    $panels.each(function panelsHidden() {
+      $(this).prop('hidden', true);
+    });
+  };
+
+  $tabsWrapper.each(function eachTabsGroup() {
+    const $this = $(this);
+    const $tablist = $('.tabs__list', $this);
+    const $tabs = $('.tabs__list-link', $this);
+    const $panels = $('.tabs__panels-item', $this);
+
+    // The tab switching function
+    const switchTab = (oldTab, newTab) => {
+      newTab.focus();
+      // Get id of panel controlled by newTab.
+      const newPanelId = $(newTab).attr('aria-controls');
+      // Make the active tab focusable by the user (Tab key) and add aria selected.
+      $(newTab).removeAttr('tabindex').attr('aria-selected', 'true');
+      $(oldTab).removeAttr('aria-selected').attr('tabindex', '-1');
+
+      // Hide all panels.
+      deactivatePanels($panels);
+      // Unhide new panel.
+      $(`#${newPanelId}`, $context).prop('hidden', false);
+    };
+
+    // Add semantics are remove user focusability for each tab
+    $tabs.each(function tabsInit(i, tab) {
+      $(tab).attr('tabindex', '-1');
+
+      // Handle clicking of tabs for mouse users
+      $(tab).on('click', (e) => {
+        e.preventDefault();
+        const $currentTab = $('[aria-selected="true"]', $tablist);
+        // Switch tab if clicked tab is not current tab.
+        if ($(e.currentTarget)[0] !== $currentTab[0]) {
+          switchTab($currentTab, e.currentTarget);
+        }
+      });
+
+      // Handle keydown events for keyboard users
+      $(tab).on('keydown', (e) => {
+        // Get the index of the current tab in the tabs node list
+        const index = $tabs.index(e.currentTarget);
+
+        // Work out which key the user is pressing and
+        // Calculate the new tab's index where appropriate
+        const { key } = e;
+        if (key === 'ArrowLeft') {
+          e.preventDefault();
+          let newTabIndex = index - 1;
+          // If first item set last items index.
+          if (index === 0) {
+            newTabIndex = $tabs.length - 1;
+          }
+          switchTab(e.currentTarget, $tabs[newTabIndex]);
+        } else if (key === 'ArrowRight') {
+          e.preventDefault();
+          let newTabIndex = index + 1;
+          // If last item set new tab index to 0.
+          if (index === $tabs.length - 1) {
+            newTabIndex = 0;
+          }
+          switchTab(e.currentTarget, $tabs[newTabIndex]);
+        } else if (key === 'ArrowDown') {
+          $panels[i].focus();
+        }
+      });
+    });
+
+    // Add tab panel semantics and hide them all.
+    $panels.each((i, panel) => {
+      $(panel).attr('tabindex', '-1');
+      $(panel).attr('aria-labelledby', $tabs[i].id);
+      $(panel).attr('hidden', true);
+    });
+
+    // Initially activate the first tab and reveal the first tab panel.
+    $($tabs[0]).removeAttr('tabindex');
+    $($tabs[0]).attr('aria-selected', 'true');
+    $($panels[0]).attr('hidden', false);
```

Comment:
Use `.first()`: `$tabs.first().removeAttr()`. Anyway, couldn't you set this in twig to start out?